src/routes/assessmentRoutes.js

```diff
@@ -2,37 +2,72 @@ const express = require("express");
 const { isLoggedIn, isInstructor } = require("../middlewares/middleware");
```

Comment:
Consider adding specific error messages or redirects in the middleware itself for consistency.